test/e2e/config/e2e_conf.yaml

```diff
@@ -59,13 +59,11 @@ providers:
     files:
     - sourcePath: "${PWD}/metadata.yaml"
       targetName: "metadata.yaml"
-    - sourcePath: "${PWD}/templates/test/cluster-template-prow-single-master.yaml"
-      targetName: "cluster-template-single-master.yaml"
-    - sourcePath: "${PWD}/templates/test/cluster-template-prow-ha.yaml"
+    - sourcePath: "${PWD}/templates/test/cluster-template-prow-ha_envsubst.yaml"
```

Comment:
You sure it is not `cluster-template-prow-ha.yaml`?

Reply:
Yes. The `cluster-template-prow-ha_envsubst.yaml` is generated by passing `cluster-template-prow-ha.yaml` through envsubst, which is done in the `e2e-substitutions` make target.

Reply:
I missed this part while reviewing, would taking e2e-substitutions into e2e-tests make target help with that, because otherwise we would have to have `_envsubst` added for each template yaml naming where we could have multiple of them in the future